test/translation/__snapshots__/transformation.spec.ts.snap

```diff
@@ -40,17 +40,10 @@ end"
 `;
 
 exports[`Transformation (classPureAbstract) 1`] = `
-"ClassB = {}
+"require(\\"lualib_bundle\\");
+ClassB = __TS__Class()
 ClassB.name = \\"ClassB\\"
-ClassB.__index = ClassB
-ClassB.prototype = {}
 ClassB.prototype.__index = ClassB.prototype
```

Comment:
What is the reason this line didn't move to __TS__Class?

Reply:
It's added conditionally, only when there are no getters. Though probably it'd be better to move it as well, an extra assignment wouldn't be a problem anyway, given how slow getters already are